ibm/mas_devops/roles/cos/tasks/providers/ibm/provision.yml

```diff
@@ -87,7 +87,7 @@
   when: cos_info.resource.id is defined
   set_fact:
     cos_resource_id: "{{ cos_info.resource.id }}"
-
+    use_hmac: "{{mas_config_scope=='app' | ternary(true,false)}}"
```

Comment:
The use_hmac should really be a top level option that is defined on the readme, rather than set when the config_scope is set to app.

Reply:
Updated as per suggestion